README.md

```diff
@@ -4,46 +4,50 @@ Our company is building a prototype ticketing system for a local burrito shop. U
 
 ## Setup
 
-* Fork this repo
-* Clone down your forked repo and change into the cloned down directory
-* Run `npm install` to install dependencies
-* Run `npm start` to start your development server
+- Fork this repo
+- Clone down your forked repo and change into the cloned down directory
+- Run `npm install` to install dependencies
+- Run `npm start` to start your development server
 
 Be sure to setup the [backend repo for Burrito Builder](https://github.com/turingschool-examples/burrito-builder-api) to be able to retrieve and save burrito orders.
 
-## Iterations 
+## Iterations
 
 ### Iteration 1
 
-Right now the app has serveral bugs in it. Run the dev server and fix whatever bugs are causing the app to crash. 
+Right now the app has several bugs in it. Run the dev server and fix whatever bugs are causing the app to crash.
 
 ### Iteration 2
 
 Connect the front end to the backend so that when the app loads, all existing orders are fetched from the server.
 
-### Iteration 3 
+### Iteration 3
 
 Add functionality to allow submission of the form when there is at least once ingredient, and a name added to the order. If one of these two things is missing, you should not be able to submit an order. If the order is submitted, a POST request should be made to the server.
```

Comment:
`least once ingredient, and`

I know you didn't change this line but there is small typo.  Should be "one".   I don't think we need the comma.